lesson6/README.md

```diff
@@ -0,0 +1,87 @@
+# Lesson 6 - How to Create an App
+
+##  Designing an app
+
```

Comment:
also, it's a bit confusing reading through this. when i started it sounds like you're giving instructions to me, so it seemed like the entire thing is homework